### ISSUE TITLE
Docs for manual installation

### ISSUE THREAD

**Issue by ianhbell** [mandeep/sublime-text-conda#14]
I tried to install manually to work around #12 , but I could not follow the instructions.  There are a lot of candidate locations, and it seems like this is the one I need: 

```
C:\Users\ihb\AppData\Roaming\Sublime Text 3\Packages\User
```

The instructions are (verbatim from README)

If you would rather install from source, simply run:

```
$  git clone git@github.com:mandeep/sublime-text-conda.git
```

inside the Sublime Text packages folder.

**Comment by mandeep** [comment]
Good point. I should add a link to Sublime Text manual package installation.

In the meantime, were you able to get the package working with that path?

**Comment by ianhbell** [comment]
It needs to get cloned into C:\Users\ihb\AppData\Roaming\Sublime Text 3\Packages

E.g.:

![image](https://user-images.githubusercontent.com/11966765/68096853-cc5dc380-fe70-11e9-831f-86ee53df8744.png)

**Comment by mandeep** [comment]
Fixed by e16941e66e1e5fc126cacad39eab8516dc32919e
